tools/run-sitl-tests.sh

```diff
@@ -7,9 +7,29 @@ if [ "$#" -ne 1 ]; then
     exit 1
 fi
 
-PX4_FIRMWARE_DIR=$1
+echo "PX4 Version is: " ${PX4_VERSION}
```

Comment:
Should that line exist? Feels like it's already done below (line 13), right?

Reply:
my bad :) .Removed it in the latest commit.